argilla-server/src/argilla_server/api/schemas/v1/suggestions.py

```diff
@@ -87,15 +87,28 @@ class SuggestionCreate(BaseSuggestion):
     ]
     agent: Optional[str] = Field(
         None,
-        regex=AGENT_REGEX,
+        pattern=AGENT_REGEX,
         min_length=AGENT_MIN_LENGTH,
         max_length=AGENT_MAX_LENGTH,
         description="Agent used to generate the suggestion",
     )
-    score: Optional[Union[float, List[float]]] = Field(
-        None,
-        min_items=SCORE_MIN_ITEMS,
-        ge=SCORE_GREATER_THAN_OR_EQUAL,
-        le=SCORE_LESS_THAN_OR_EQUAL,
-        description="The score assigned to the suggestion",
-    )
+    score: Optional[Union[float, List[float]]] = Field(None, description="The score assigned to the suggestion")
```

Comment:
Instead of adding custom validations maybe we can define custom types like `SuggestionScore`. 

Take a look to what I did here: https://github.com/argilla-io/argilla/pull/5508/files#diff-a9277ada64762a8062c1e132567edba6ce2aec3e279d1ded0452b9557bfd6b26